datasets/usda_nass_agriculture/pipelines/usda_nass_agriculture/pipeline.yaml

```diff
@@ -0,0 +1,80 @@
+                       wget -O /home/airflow/gcs/data/usda_nass_agriculture/raw_files/economics.txt.gz ftp://ftp.nass.usda.gov/quickstats/qs.economics_20221129.txt.gz ;"
+
+    - operator: "BashOperator"
+      description: "Unzip data"
+      args:
+        task_id: "bash_unzip"
+        bash_command: "gunzip -f -v -k /home/airflow/gcs/data/usda_nass_agriculture/raw_files/census2002.txt.gz ;
+                       gunzip -f -v -k /home/airflow/gcs/data/usda_nass_agriculture/raw_files/census2007.txt.gz ;
+                       gunzip -f -v -k /home/airflow/gcs/data/usda_nass_agriculture/raw_files/census2012.txt.gz ;
+                       gunzip -f -v -k /home/airflow/gcs/data/usda_nass_agriculture/raw_files/animal_products.txt.gz ;
+                       gunzip -f -v -k /home/airflow/gcs/data/usda_nass_agriculture/raw_files/economics.txt.gz ;"
+
+    - operator: "KubernetesPodOperator"
+      description: "ETL within the kubernetes pod"
+      args:
+        task_id: "kub_csv_transform"
+        startup_timeout_seconds: 1000
+        name: "load_data"
+        namespace: "composer"
+        service_account_name: "datasets"
+        image_pull_policy: "Always"
+        image: "{{ var.json.usda_nass_agriculture.container_registry.run_csv_transform_kub }}"
+        env_vars:
+          SOURCE_GCS_PATH: "data/usda_nass_agriculture/raw_files/"
+          DESTINATION_GCS_PATH: "data/usda_nass_agriculture/transformed_files/"
+          PROJECT_ID: "{{ var.value.gcp_project }}"
+          DATASET_ID: "usda_nass_agriculture"
+          GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          SCHEMA_FILEPATH: "schema.json"
+          RENAME_MAPPINGS: '{"SOURCE_DESC": "source_desc", "SECTOR_DESC": "sector_desc", "GROUP_DESC": "group_desc", "COMMODITY_DESC": "commodity_desc", "CLASS_DESC": "class_desc", "PRODN_PRACTICE_DESC": "prodn_practice_desc", "UTIL_PRACTICE_DESC": "util_practice_desc", "STATISTICCAT_DESC": "statisticcat_desc", "UNIT_DESC": "unit_desc", "SHORT_DESC": "short_desc", "DOMAIN_DESC": "domain_desc", "DOMAINCAT_DESC": "domaincat_desc", "AGG_LEVEL_DESC": "agg_level_desc", "STATE_ANSI": "state_ansi", "STATE_FIPS_CODE": "state_fips_code", "STATE_ALPHA": "state_alpha", "STATE_NAME": "state_name", "ASD_CODE": "asd_code", "ASD_DESC": "asd_desc", "COUNTY_ANSI": "county_ansi", "COUNTY_CODE": "county_code", "COUNTY_NAME": "county_name", "REGION_DESC": "region_desc", "ZIP_5": "zip_5", "WATERSHED_CODE": "watershed_code", "WATERSHED_DESC": "watershed_desc", "CONGR_DISTRICT_CODE": "congr_district_code", "COUNTRY_CODE": "country_code", "COUNTRY_NAME": "country_name", "LOCATION_DESC": "location_desc", "YEAR": "year", "FREQ_DESC": "freq_desc", "BEGIN_CODE": "begin_code", "END_CODE": "end_code", "REFERENCE_PERIOD_DESC": "reference_period_desc", "WEEK_ENDING": "week_ending", "LOAD_TIME": "load_time", "VALUE": "value", "CV_%": "cv_percent"}'
```

Comment:
Please remove single quote, and newline each section so that it is not long.  Thanks!

Ie.

RENAME_MAPPINGS: >-
    {
        "SOURCE_DESC": "source_desc", 
        "SECTOR_DESC": "sector_desc",
        ...
    }

Reply:
Done